nixos/modules/programs/steam.nix

```diff
@@ -29,6 +30,23 @@ in {
       '';
     };
 
+    env = mkOption {
```

Comment:
Why does steam need local env vars? Why not use an existing NixOS option to set env vars globally?

Reply:
See the example; these are env vars you might want to set for all games but not system-wide.